### ISSUE TITLE
nodetool repair failed, while creating a new CF folder

### ISSUE THREAD

**Issue by noamha** [scylladb/scylla#1596]
*Installation details*
Scylla version (or git commit hash):
1.3.rc2-20160808.b088dd7.el7.centos
Cluster size: 3 Scylla nodes, i2.8xlarge
Data size: 80GB per node, 10M keys, 5 columns per key, 1.5K value size
Scylla Version: 1.3.rc2-20160808.b088dd7.el7.centos

After c-s finished writing the data, I've stopped one of the nodes, deleted 40GB out of the 80GB of data, started it and ran "nodetool repair"

After a few minutes I've received the following error:
```
[2016-08-17 17:18:29,867] Starting repair command #10, repairing 1 ranges for keyspace keyspace1 (parallelism=SEQUENTIAL, full=true)

[2016-08-17 17:27:21,060] Repair session 10 failed
[2016-08-17 17:27:21,060] Repair session 10 finished
[2016-08-17 17:27:21,156] Starting repair command #11, repairing 1 ranges for keyspace system_traces (parallelism=SEQUENTIAL, full=true)
[2016-08-17 17:27:21,259] Repair session 11 finished
[2016-08-17 17:27:21,395] Starting repair command #12, repairing 1 ranges for keyspace system (parallelism=SEQUENTIAL, full=true)
[2016-08-17 17:27:21,497] Repair session 12 finished
error: nodetool failed, check server logs
-- StackTrace --
java.lang.RuntimeException: nodetool failed, check server logs
       at org.apache.cassandra.tools.NodeTool$NodeToolCmd.run(NodeTool.java:290)
       at org.apache.cassandra.tools.NodeTool.main(NodeTool.java:202)
```

Some journalctl output:
```
Aug 17 17:27:21 ip-172-30-0-15 scylla[13875]:  [shard 0] repair - [repair #dabf8125-649f-11e6-8a26-000000000004] new session: will sync {} on range (9087044348664198083, 9106815045295896443] for system.{size_estimates, sstable_activity, compaction_history, compactions_
Aug 17 17:27:21 ip-172-30-0-15 scylla[13875]:  [shard 0] repair - [repair #dabfa830-649f-11e6-8a26-000000000004] new session: will sync {} on range (9106815045295896443, 9116814397968294034] for system.{size_estimates, sstable_activity, compaction_history, compactions_
Aug 17 17:27:21 ip-172-30-0-15 scylla[13875]:  [shard 0] repair - [repair #dabfa831-649f-11e6-8a26-000000000004] new session: will sync {} on range (9116814397968294034, +inf) for system.{size_estimates, sstable_activity, compaction_history, compactions_in_progress, ra
Aug 17 17:27:21 ip-172-30-0-15 scylla[13875]:  [shard 0] repair - repair 12 completed sucessfully
```

It also created a second CF folder stanrdard1 folder which was empty.
The original standard1 folder did not grow in size and stayed at 40GB

I've used the following yaml file to run using scylla-cluster-test:
```
cassandra_row_limit: 10000000
cassandra_column_per_row: 5
# default cassandra stress duration (min) if none specified
cassandra_stress_duration: 15
# default cassandra stress thread number if none specified
cassandra_stress_threads: 200
# limit
cassandra_stress_limits: 5000/s
# cassandra-stress row size
cassandra_stress_row_size: 1500
# default cassandra stress population size if none specified
cassandra_stress_population_size: 1000000000
n_db_nodes: 3
# If you want to use more than 1 loader node, I recommend
# increasing the size of the DB instance (instance_type_db parameter),
# since 't2.micro' nodes tend to struggle with a lot of load.
n_loaders: 4
# Number of monitor nodes running prometheus
n_monitor_nodes: 1
# From 0.19 on, iotune will require bigger disk, so let's use a big
# loader instance by default.
instance_type_loader: 'c3.xlarge'
# Size of AWS monitor instance
instance_type_monitor: t2.small
# Nemesis class to use (possible types in sdcm.nemesis). Example: StopStartMonkey
nemesis_class_name: 'DoNothingMonkey'
# Nemesis sleep interval to use if None provided specifically
nemesis_interval: 30
# Prefix for your AWS VMs (handy for telling instances from different
# users apart). If you leave this empty, the prefix will be your unix username.
user_prefix:
# Failure/post test behavior
# Default: Destroy AWS instances and credentials (destroy)
# Keep AWS instances running and leave credentials alone (keep)
# Stop AWS instances and leave credentials alone (stop)
failure_post_behavior: keep
# Space node threshold before starting nemesis (bytes)
# The default value is 6GB (6x1024^3 bytes)
# This value is supposed to reproduce
# https://github.com/scylladb/scylla/issues/1140
space_node_threshold: 10000

# The new db binary will be uploaded to db instance to replace the one
# provided by the ami. This is useful to test out a new scylla version
# without making a new ami.
# update_db_binary: $path_of_the_new_scylla_binary
update_db_binary: ''

backends: !mux
    libvirt: !mux
        cluster_backend: 'libvirt'
        libvirt_uri: 'qemu:///system'
        libvirt_bridge: 'virbr0'
        scylla_repo: 'http://downloads.scylladb.com/rpm/centos/scylla-1.2.repo'
        libvirt_loader_image: '/path/to/your/loader_base_image.qcow2'
        libvirt_loader_image_user: 'centos'
        libvirt_loader_image_password: '123456'
        libvirt_loader_os_type: 'linux'
        libvirt_loader_os_variant: 'centos7.0'
        libvirt_loader_memory: 2048
        libvirt_db_image: '/path/to/your/db_base_image.qcow2'
        libvirt_db_image_user: 'centos'
        libvirt_db_image_password: '123456'
        libvirt_db_os_type: 'linux'
        libvirt_db_os_variant: 'centos7.0'
        libvirt_db_memory: 2048
        libvirt_monitor_image: '/path/to/your/monitor_base_image.qcow2'
        libvirt_monitor_image_user: 'centos'
        libvirt_monitor_image_password: '123456'
        libvirt_monitor_os_type: 'linux'
        libvirt_monitor_os_variant: 'centos7.0'
        libvirt_monitor_memory: 2048
    aws: !mux
        # What is the backend that the suite will use to get machines from.
        cluster_backend: 'aws'
        # From 0.19 on, iotune will require bigger disk, so let's use a big
        # loader instance by default.
        # Size of AWS monitor instance
        us_west_1:
            region_name: 'us-west-1'
            security_group_ids: 'sg-dcd785b9'
            subnet_id: 'subnet-10a04c75'
            ami_id_db_scylla: 'ami-e2246382'
            ami_db_scylla_user: 'centos'
            ami_id_loader: 'ami-e2246382'
            ami_loader_user: 'centos'
            ami_id_db_cassandra: 'ami-3cf7c979'
            ami_db_cassandra_user: 'ubuntu'
            ami_id_monitor: 'ami-e2246382'
            ami_monitor_user: 'centos'
        us_west_2:
            region_name: 'us-west-2'
            security_group_ids: 'sg-81703ae4'
            subnet_id: 'subnet-5207ee37'
            ami_id_db_scylla: 'ami-ed4d8b8d'
            ami_db_scylla_user: 'centos'
            ami_id_loader: 'ami-ed4d8b8d'
            ami_loader_user: 'centos'
            ami_id_db_cassandra: 'ami-1cff962c'
            ami_db_cassandra_user: 'ubuntu'
            ami_id_monitor: 'ami-ed4d8b8d'
            ami_monitor_user: 'centos'
        us_east_1:
            region_name: 'us-east-1'
            security_group_ids: 'sg-c5e1f7a0'
            subnet_id: 'subnet-ec4a72c4'
            ami_id_db_scylla: 'ami-09a63a1e'
            ami_db_scylla_user: 'centos'
            ami_id_loader: 'ami-dd46ffca'
            ami_loader_user: 'centos'
            ami_id_db_cassandra: 'ami-ada2b6c4'
            ami_db_cassandra_user: 'ubuntu'
            ami_id_monitor: 'ami-dd46ffca'
            ami_monitor_user: 'centos'

databases: !mux
    cassandra:
        db_type: cassandra
        instance_type_db: 'm3.large'
    scylla:
        db_type: scylla
        # Let's use c3.large since we're using iotune
        # and we'll stress the DB nodes more thoroughly
        instance_type_db: 'i2.8xlarge'
```

**Comment by slivne** [comment]
@noamha - I think the issue maybe the fact that you are running multiple loaders and they are starting at the same time and creating the scehams via different nodes in parallel

Can you check in your log the time the cassandra-stress command started running

Can you check that you have the following commit in scylla-cluster-tests scylladb/scylla-cluster-tests@88c4a4a41ab8755a6204962e5a18ce8be2da1193 - I think it solved the issue for me.

**Comment by noamha-ken** [comment]
@slivne I have it now, I'll try with the new commit.
Also, just to be safe if it doesn't work I'll try with a single loader

Thanks.

**Comment by noamha-ken** [comment]
@slivne looks like it's solved, repair completed successfuly.
Thanks

**Comment by slivne** [comment]
duplicate of #420
